Added arm64 version

dawidkulas opened the pull request:
This is an Arm64 version for usage with Raspberry Pi or any other arm64 based devices.
I have tested VNC on Raspberry Pi 4 Model B with 4 GB of RAM and it worked fine.
Mounting ISO and screenshots haven't been tested.

Closes #4

FoxxMD commented:
This can also be done without needing to specify specific architectures. Replacing the prerequisite package lines and `wget` for the specific debs with this:

```shell
apt-get install -y wget software-properties-common libx11-dev gcc xdotool gnupg ca-certificates curl && \
    curl -s https://repos.azul.com/azul-repo.key | gpg --dearmor -o /usr/share/keyrings/azul.gpg && \
    echo "deb [signed-by=/usr/share/keyrings/azul.gpg] https://repos.azul.com/zulu/deb stable main" | tee /etc/apt/sources.list.d/zulu.list && \
    apt-get update && \
    apt-get install zulu8-jre -y
```

(remove `DOCKER_IMAGE_PLATFORM=arm64 as well)

I'm doing this in my [fork](https://github.com/FoxxMD/docker-avocent/blob/master/Dockerfile) and publishing working [cross platform images with buildx](https://hub.docker.com/repository/docker/foxxmd/avocent-vkvm/tags?page=1&ordering=last_updated)

dawidkulas commented:
@FoxxMD Thank you very much for the advice! I added the repository as described on Azul website, made that additional dependencies are removed when not needed and also remove wget from being downloaded as it is not needed anymore.

And as you said, I removed ```DOCKER_IMAGE_PLATFORM=arm64``` to make the build truly cross-platform.

dawidkulas commented:
Sorry, I made a mistake of removing wget and then I realised that it is required. It is now fixed.

FoxxMD commented:
Additional improvement that would be beneficial for all images @DomiStyle ...

In this commit/PR https://github.com/DomiStyle/docker-idrac6/pull/49 java.security was removed in order to "un"disable TLSv1.1 but it also made the fix architecture specific:

RUN rm /usr/lib/jvm/zulu-7-**amd64**/jre/lib/security/java.security

This can be achieved without architecture specific steps by overriding java.security with a new file in the java exec arg:

1. Remove `RUN rm /usr/lib/jvm/zulu-7-**amd64**/jre/lib/security/java.security`
2. create a [`java.security` file](https://github.com/FoxxMD/docker-avocent/blob/master/ikvm.java.security) that overrides the affected settings
3.  In dockerfile [`COPY java.security /java.security`](https://github.com/FoxxMD/docker-avocent/blob/master/Dockerfile#L33C1-L33C44)
4. In [`startapp.sh`](https://github.com/FoxxMD/docker-avocent/blob/master/startapp.sh#L121) add the java arg `-Djava.security.properties="/java.security"`
